http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20230614_add_app_controlled_resource.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="leonardo" author="rtitle" id="add_app_controlled_resource_table">
+        <createTable tableName="APP_CONTROLLED_RESOURCE">
```

Comment:
We had a `RUNTIME_CONTROLLED_RESOURCE` table but no `APP_CONTROLLED_RESOURCE` (probably because until now apps didn't provision any WSM resources).